Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
+
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
`DOMContentLoaded` is fired when the document has been completely loaded and parsed (no stylesheets, images etc.). `load` is fired when all stuff (images, subframes etc.) have finished loading.

I've tried with `load` but it doesn't work, helpers are not registered corectly.